cmd/configtxgen/main.go

```diff
@@ -220,7 +228,7 @@ func main() {
 	flag.StringVar(&outputBlock, "outputBlock", "", "The path to write the genesis block to (if set)")
 	flag.StringVar(&channelID, "channelID", "", "The channel ID to use in the configtx")
 	flag.StringVar(&outputChannelCreateTx, "outputCreateChannelTx", "", "The path to write a channel creation configtx to (if set)")
-	flag.StringVar(&channelCreateTxBaseProfile, "channelCreateTxBaseProfile", "", "Specifies a profile to consider as the orderer system channel current state to allow modification of non-application parameters during channel create tx generation. Only valid in conjunction with 'outputCreateChannelTx'.")
+	flag.StringVar(&channelCreateTxBaseProfile, "channelCreateTxBaseProfile", "", "[DEPRECATED] Specifies a profile to consider as the orderer system channel current state to allow modification of non-application parameters during channel create tx generation. Only valid in conjunction with 'outputCreateChannelTx'.")
```

Comment:
I think we can remove this flag and we also need to run the below script to update the docs:

https://github.com/hyperledger/fabric/blob/main/scripts/help_docs.sh

Reply:
I think we want to maintain backwards compatibility? i.e. be able to manage with the new version of the tool old deployments?

If this is not the case we can do a much more drastic pruning, but the idea initially was to be conservative and do a minimal effort change.

Reply:
`outputCreateChannelTx` this flag is also deprecated, we no longer need channel creation TXs in v3

Reply:
also the inspect CreateChannelTx, and the respective entries in the *.md file below. everything about channel create TX is deprecated.